lesson8/main.py

```diff
@@ -0,0 +1,73 @@
+import json
+from user import User
+
+
+def add_user(db, us):
+    db[us.name] = us.password
+
+
+def check_name(db):
+    while True:
+        username = input("Enter your username for a registration.\n")
+        if username in db.keys():
```

Comment:
So.. I see db where.. and if I try to call check_name after defining this function, I will get an error..
We have to write the code in such a way that the definition of our variables and constants is on top